pcmanfm/desktopwindow.cpp

```diff
@@ -532,7 +532,7 @@ QImage DesktopWindow::loadWallpaperFile(QSize requiredSize) {
 
         // read info file
         QString origin;
-        info.setFileName(cacheFileName % ".info");
+        info.setFileName(cacheFileName + QStringLiteral(".info"));
```

Comment:
The are still other cases of ` % ` in `desktopwindow.cpp` and `settings.cpp`. Now that you've replaced them in some places, wouldn't it be better to replace them completely and then remove `#include <QStringBuilder>` from:
```
desktopwindow.cpp
preferencesdialog.cpp
settings.cpp
```